internal/sdkprovider/service/account/account_team.go

```diff
@@ -41,7 +40,7 @@ var aivenAccountTeamSchema = map[string]*schema.Schema{
 
 func ResourceAccountTeam() *schema.Resource {
 	return &schema.Resource{
-		Description:   "The Account Team resource allows the creation and management of an Account Team.",
+		Description:   `Creates and manages a team.`,
```

Comment:
Could you please set the property `Deprecated` there and fill it with information as well? And in other resources, too.